tests/data/placebo/test_kms_post_finding/kms.ListKeys_1.json

```diff
@@ -0,0 +1,93 @@
+{
+    "status_code": 200,
+    "data": {
+        "Keys": [
+            {
+                "KeyId": "09893686-eeda-4bbb-aa45-c0f7560fa6ff",
```

Comment:
There might be an opportunity to trim the keys here since not checking length and only checking one key, e.g. 44d25a5c-7efa-44ed-8436-b9511ea921b3

Reply:
right thats what the commented section is kms was about.